src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -88,6 +64,9 @@ const NavHeader = (props) => {
 
   const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
 
+  const TopicDefaultIcon =
+    followType === 'incognito' ? TopicAnonDefaultIcon : TopicSignedDefaultIcon;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This conditional assignment of `TopicDefaultIcon` based on `followType` is a good way to switch between different icons. Just ensure that the `followType` prop is always provided or has a default value to prevent any potential issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->